testing/typescript/data/data_source.ts

```diff
@@ -0,0 +1,8 @@
+import DataSource from '../../../js/data/data_source';
+
+export function promiseResolveAcceptsMultipleArguments() {
+  const callback: (a: any, b: any) => void = undefined;
+  const dataSource = new DataSource({});
```

Comment:
```suggestion
  const callback: (a: any, b: any) => void = ANY;
  const dataSource: DataSource<any, any> = ANY;
```
We have to share this `ANY` between files